print-bgp.c

```diff
@@ -2543,6 +2559,76 @@ bgp_attr_print(netdissect_options *ndo,
             len -= 12;
         }
         break;
+    case BGPTYPE_BGPSEC_PATH:
+    {
+        uint16_t sblen, splen;
+
+        splen = GET_BE_U_2(tptr);
+
+        /*
+         * A secure path has a minimum length of 8 bytes:
+         * 2 bytes length field
+         * 6 bytes per secure path segment
+         */
+        if (splen < 8) {
```

Comment:
To harmonize with other changes these four lines could be replaced with:
`ND_ICHECKMSG_U("secure path length", splen, <, 8);`